### PR TITLE
EOS-23261: Single node VM Deployment using factory+field commands fai…

### PR DESCRIPTION
**Pull request by tanujashinde0405**
…ling for HA deployment

Signed-off-by: Tanuja Shinde <tanuja.shinde@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: lr-cli/cortx_setup/commands/cluster/config/component.py [hunk 1]
```diff
@@ -90,22 +92,32 @@ def run(self, component_group):
...
-                self.logger.debug(f"Deploying cortx ha components on {secondaries}")
-                self._configure(
-                    ['ha.cortx-ha'],
-                    stages=[
-                        'config.post_install', 'config.prepare',
-                        'config.config', 'config.init_mod'],
-                    targets=secondaries
-                )
+                if node_count > 1:
+                    self.logger.debug(f"Deploying cortx ha components on {primary}")
```
Comment (83bhp) [inline lr-cli/cortx_setup/commands/cluster/config/component.py:96]: The deployment block for primary can be kept out of "if" check and have that check only for secondary nodes. This way code block for primary will not be duplicated.

**Review by nitin-seagate — CHANGES_REQUESTED** [review]
Please make the if-block only for secondaries (whenever node_count > 1)

**Comment by tanujashinde0405** [comment]
> Please make the if-block only for secondaries (whenever node_count > 1)

Done

**Review by 83bhp — APPROVED** [review]

**Pull request merged**